python/_restclient/tests/test_restclient.py

```diff
@@ -141,3 +141,12 @@ def test_get_headers_have_precedent_over_instance(basic_test_server):
 
```

Comment:
Both `test_restclient.py` and `test_client.py` are failing on my machine with: "fixture 'aiohttp_raw_server' not found"

Reply:
Similar issue with `test_nwis.py`